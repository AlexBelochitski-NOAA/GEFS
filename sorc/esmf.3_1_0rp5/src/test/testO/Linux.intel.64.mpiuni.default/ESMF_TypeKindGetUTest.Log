20190610 112230.751964 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112230.752208 INFO     PET0  Beginning Test, file ESMF_TypeKindGetUTest.F90, line          74
20190610 112230.752225 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112230.752247 INFO     PET0  PASS   Getting ESMF_TypeKind parameter of default integer, ESMF_TypeKindGetUTest.F90, line          83
20190610 112230.752261 INFO     PET0  PASS   Getting ESMF_TypeKind parameter of ESMF_KIND_I1 integer, ESMF_TypeKindGetUTest.F90, line          94
20190610 112230.754102 INFO     PET0  PASS  Checking ESMF_TypeKind parameter for ESMF_KIND_I1 integer, ESMF_TypeKindGetUTest.F90, line         105
20190610 112230.754114 INFO     PET0  PASS   Getting ESMF_TypeKind parameter of ESMF_KIND_I2 integer, ESMF_TypeKindGetUTest.F90, line         118
20190610 112230.754123 INFO     PET0  PASS  Checking ESMF_TypeKind parameter for ESMF_KIND_I2 integer, ESMF_TypeKindGetUTest.F90, line         129
20190610 112230.754132 INFO     PET0  PASS   Getting ESMF_TypeKind parameter of ESMF_KIND_I4 integer, ESMF_TypeKindGetUTest.F90, line         141
20190610 112230.754141 INFO     PET0  PASS  Checking ESMF_TypeKind parameter for ESMF_KIND_I4 integer, ESMF_TypeKindGetUTest.F90, line         147
20190610 112230.754233 INFO     PET0  PASS  Getting ESMF_TypeKind parameter of ESMF_KIND_I8 integer, ESMF_TypeKindGetUTest.F90, line         155
20190610 112230.754244 INFO     PET0  PASS  Checking ESMF_TypeKind parameter for ESMF_KIND_I8 integer, ESMF_TypeKindGetUTest.F90, line         161
20190610 112230.754253 INFO     PET0  PASS  Getting ESMF_TypeKind parameter of  real, ESMF_TypeKindGetUTest.F90, line         173
20190610 112230.754265 INFO     PET0  PASS  Getting ESMF_TypeKind parameter of ESMF_KIND_R4 real, ESMF_TypeKindGetUTest.F90, line         183
20190610 112230.754274 INFO     PET0  PASS  Checking ESMF_TypeKind parameter for ESMF_KIND_R4 real, ESMF_TypeKindGetUTest.F90, line         189
20190610 112230.754283 INFO     PET0  PASS  Getting ESMF_TypeKind parameter of ESMF_KIND_R8 real, ESMF_TypeKindGetUTest.F90, line         197
20190610 112230.754292 INFO     PET0  PASS  Checking ESMF_TypeKind parameter for ESMF_KIND_R8 real, ESMF_TypeKindGetUTest.F90, line         203
20190610 112230.754306 INFO     PET0  Number of failed tests:           0
20190610 112230.754312 INFO     PET0  Ending Test, file ESMF_TypeKindGetUTest.F90, line         205
