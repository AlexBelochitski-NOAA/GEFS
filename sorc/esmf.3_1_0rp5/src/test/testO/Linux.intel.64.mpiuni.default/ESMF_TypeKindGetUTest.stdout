  Beginning Test, file ESMF_TypeKindGetUTest.F90, line          74
  NUMBER_OF_PROCESSORS           1
  PASS   Getting ESMF_TypeKind parameter of default integer, ESMF_TypeKindGetUTe
 st.F90, line          83
 The typekind of default integer is           3
  PASS   Getting ESMF_TypeKind parameter of ESMF_KIND_I1 integer, ESMF_TypeKindG
 etUTest.F90, line          94
  PASS  Checking ESMF_TypeKind parameter for ESMF_KIND_I1 integer, ESMF_TypeKind
 GetUTest.F90, line         105
  PASS   Getting ESMF_TypeKind parameter of ESMF_KIND_I2 integer, ESMF_TypeKindG
 etUTest.F90, line         118
  PASS  Checking ESMF_TypeKind parameter for ESMF_KIND_I2 integer, ESMF_TypeKind
 GetUTest.F90, line         129
  PASS   Getting ESMF_TypeKind parameter of ESMF_KIND_I4 integer, ESMF_TypeKindG
 etUTest.F90, line         141
  PASS  Checking ESMF_TypeKind parameter for ESMF_KIND_I4 integer, ESMF_TypeKind
 GetUTest.F90, line         147
  PASS  Getting ESMF_TypeKind parameter of ESMF_KIND_I8 integer, ESMF_TypeKindGe
 tUTest.F90, line         155
  PASS  Checking ESMF_TypeKind parameter for ESMF_KIND_I8 integer, ESMF_TypeKind
 GetUTest.F90, line         161
  PASS  Getting ESMF_TypeKind parameter of  real, ESMF_TypeKindGetUTest.F90, lin
 e         173
 The typekind of real is           5
  PASS  Getting ESMF_TypeKind parameter of ESMF_KIND_R4 real, ESMF_TypeKindGetUT
 est.F90, line         183
  PASS  Checking ESMF_TypeKind parameter for ESMF_KIND_R4 real, ESMF_TypeKindGet
 UTest.F90, line         189
  PASS  Getting ESMF_TypeKind parameter of ESMF_KIND_R8 real, ESMF_TypeKindGetUT
 est.F90, line         197
  PASS  Checking ESMF_TypeKind parameter for ESMF_KIND_R8 real, ESMF_TypeKindGet
 UTest.F90, line         203
  Number of failed tests:           0
  Ending Test, file ESMF_TypeKindGetUTest.F90, line         205
